index.js

```diff
@@ -0,0 +1,41 @@
+// Add/Remove Added Books
+import { Book, Store } from './modules/storage.js';
+import UI from './modules/ui.js';
+import {
+  tableSection, addSection, contactSection, listLink, addLink, contactLink,
+} from './modules/spa.js';
+import { DateTime } from './modules/luxon.js';
+// Add EventListening
+document.querySelector('form').addEventListener('submit', (e) => {
+  e.preventDefault();
+  const title = document.querySelector('#title').value;
+  const author = document.querySelector('#author').value;
+  const book = new Book(title, author);
+  UI.addBookToList(book);
+  Store.addBook(book);
+  UI.clearFields();
+});
+document.querySelector('.book-list').addEventListener('click', (e) => {
+  UI.deleteBook(e.target);
```

Comment:
- [ ] It will be better if you add a function to delete the book from local storage like you did when you add the book.